apps/dg/english.lproj/function_strings.json

```diff
@@ -432,12 +432,12 @@
       {
         "name": "\"otherDataSet\"",
         "type": "string",
-        "description": "The data set in which you want to look up the value."
+        "description": "The data set in which you want to look up the value. Note that the string needs to be inside quotes."
```

Comment:
I suggest:
The name of the data set (in quotes) in which you want to look up the value.